CONTRIBUTING.md

```diff
@@ -64,6 +60,18 @@ If you want to propose an idea or a feature and work on it, create a new issue i
 
 You should expect someone to comment on the issue within 48 hours after it is created. If the proposal in the issue is accepted, you should then follow the process for `Working on current tasks` above.
 
+# Code review process
+
+We have two groups of code reviewers:  Super owners and normal owners.  When a PR is created:
+
+- a super owner will be automatically assigned to review.
+- they might choose to review themself or delegate to someone else who belongs either to the super owners or the normal owners group.
+- the delegate will perform the review and as needed engage other reviewers as well.  They will review your tests, and make sure that they can convince themselves the test coverage is adequate before they even look into the change, so make sure you tested all the corner cases.
+
+The author is also free to directly request reviews from specific persons [through the github ui](https://docs.github.com/en/github/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/requesting-a-pull-request-review).  In this case, a super owner will still be asked to review the PR and their job will be to check if the already reviewer is sufficient or additional reviewers are needed.
+
+If you are interested in becoming a code reviewer, please get in touch with us on zulip.
```

Comment:
Question is: who is this documented for?  If this is for contributors from outside of NEAR than I don’t think they care about owners etc.  It’d be enough to just say that someone will be automatically assigned to the review.  I’ve contributed to many free software projects and not once I really cared about code ownership structure.

Reply:
This is documented for everyone contributing to NEAR regardless of their employment status.

Reply:
That's a good question! 

The primary audience here is maintainers and prospective maintainers. It's true that 99% of contributors don't care about this. But there's this 1% contributor who could potentially become a maintainer, and we care a lot about them. By putting this into out there, we lure unsuspecting contributors into the maintainer role :) 

It's also useful to have this policy documented *somewhere* so people who are responsible for doing reviews know how that works. And I'd bias towards making this public, because why not? 


At the same time, yeah, this probably don't deserve prime-time attention from the readers, so it might be better to move this to the back of the document. The relevant section of rust-analyzer's doc is reliberately the last one: 

https://github.com/rust-lang/rust-analyzer/tree/master/docs/dev#permissions

(if you manage to read the contributing.md to the end, you are at risk of becoming a maintainer).

Reply:
moved to the bottom of the doc.